pypeit/core/flexure.py

```diff
@@ -847,25 +849,52 @@ def spec_flexure_slit_global(sciImg, waveimg, global_sky, par, slits, slitmask,
     return flex_list
 
 
-def get_archive_spectrum(sky_file):
+def get_archive_spectrum(sky_file, obj_skyspec=None, spec_fwhm_pix=None):
     """ Load an archival sky spectrum
 
     Args:
         sky_file (:obj:`str`):
-            Sky file
+            Name of the archival sky file. If equal to 'model', instead,
+            a model sky spectrum will be generated using see :func:`~pypeit.wavemodel.nearIR_modelsky`.
+        obj_skyspec (`linetools.spectra.xspectrum1d.XSpectrum1d`_, optional):
```

Comment:
Is it correct that this must be set if `sky_file == "model"`

if so, please note it.